docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -228,6 +228,9 @@ implementation is used:
   - Number of threads used for retrieving metadata. Currently, only table loading 
     is parallelized.
   - `8`
+* - `iceberg.file-delete-threads`
+  - Number of threads to use for deleting files when running `expire_snapshots` procedure.
+  - Double the number of processors on the coordinator node.
```

Comment:
Misc question: Do we have any way to restrict certain settings only for the coordinator ?

Outside of the scope of the current PR